tests/promise/promise.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, CKEDITOR, window */
+
+'use strict';
+
+describe( 'Promise', function() {
```

Comment:
Of course, the above is valid only if `CKEDITOR.require` is used outside `it()` - that's why I mentioned this aspect too.